wandb/sdk/artifacts/artifact_saver.py

```diff
@@ -72,9 +72,9 @@ def save(
         incremental: bool = False,
         history_step: Optional[int] = None,
         base_id: Optional[str] = None,
-    ) -> Optional[Dict]:
+    ) -> Tuple[Optional[Dict], Optional[concurrent.futures.Future]]:
```

Comment:
Nit: let's be more specific
```suggestion
    ) -> Tuple[Optional[Dict], Optional[concurrent.futures.Future[None]]]:
```

Reply:
Ah, this only works in Python 3.9+.